component/main.jsonnet

```diff
@@ -4,7 +4,164 @@ local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.openshift4_s3_forwarder;
+local app_name = inv.parameters._instance;
+
+local serviceaccount = kube.ServiceAccount(app_name);
+
+local configmap = kube.ConfigMap(app_name) {
+  data: {
+    'fluentd-loglevel': params.fluentd.loglevel,
+    's3-bucket': params.s3.bucket,
+    's3-endpoint': params.s3.endpoint,
+    's3-interval': params.s3.interval,
+    'td-agent.conf': |||
```

Comment:
Is this static enough or should it be defined in the defaults.yaml to be overrideable?

Reply:
These are all define in the defaults.
Well, I could create a general fluentd component...

Reply:
I'll create a ticket, to look into how i could pack this in a fluentd component without loosing the special configuration for splunk or s3 👍